packages/url_launcher/url_launcher_linux/lib/url_launcher_linux.dart

```diff
@@ -1,3 +0,0 @@
-// The url_launcher_platform_interface defaults to MethodChannelUrlLauncher
```

Comment:
Not needed anymore?

Reply:
Correct, publishing no longer warns on having no Dart code. I figured it was easier to just remove given that.

Reply:
Absolutely, this is great!